plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/quantity-selector/block.json

```diff
@@ -11,6 +11,23 @@
 	"supports": {
 		"interactivity": true
 	},
+	"styles": [
+		{
+			"name": "input",
+			"label": "Input",
+			"isDefault": false
+		},
+		{
+			"name": "stepper",
+			"label": "Stepper",
+			"isDefault": true
+		},
+		{
+			"name": "hidden",
+			"label": "Hidden",
+			"isDefault": false
+		}
+	],
```

Comment:
The _Quantity Selector_ block is only used inside the _Add to Cart + Options_ block, in which case we don't support the _Input_ style, and _Hidden_ can be accomplished by removing the block. So I think we should remove these lines?